code/Model/EditableFormField/EditableFileField.php

```diff
@@ -196,7 +196,7 @@ public function validate()
 
 
 
-    public function getFormField()
+    public function getFormField(bool $protectFolder = false): FileField
```

Comment:
I know this is still in draft so if this is already something you're aware of, just ignore me:
We can't make an API change like this one in a major release - this will break things for anyone who has a subclass of `EditableFileField` where they've implemented their own `getFormField()` method.